src/main/scss/templates/_friends.scss

```diff
@@ -0,0 +1,53 @@
+// Friends styling
+.friends {
+  display: block;
+  padding: $size-base * 2;
+
+  .card {
+    background-color: $color-white;
+    padding: $size-base * 2;
+    border-radius: $size-base;
+    margin-right: $size-base * 3;
+    margin-bottom: $size-base * 3;
+    text-align: center;
+    width: 250px;
+    height: 300px;
+
+    .username {
```

Comment:
Consider setting the spacing between the username and image the same spacing between the top of the card and the top of the image.